dali/python/dummy/setup.py.in

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools.command.install import install
+from setuptools import setup, find_packages
+import os
+
+class CustomInstallCommand(install, object):
+    def run(self):
+        new_path = os.path.join('https://developer.download.nvidia.com/compute/redist', '@DALI_FLAVOR_MINUS@')
+        raise NameError( \
+"""\n###########################################################################################################
+ERROR: Package nvidia-dali@DALI_FLAVOR_MINUS@ is now deprecated. Please install nvidia-dali@DALI_FLAVOR_MINUS@-cuda@CUDA_VERSION_SHORT_DIGIT_ONLY@ instead:
+pip install --extra-index-url {} nvidia-dali@DALI_FLAVOR_MINUS@-cuda@CUDA_VERSION_SHORT_DIGIT_ONLY@
+For more information, go to https://docs.nvidia.com/deeplearning/dali/user-guide/docs/installation.html#id1
```

Comment:
The link is now `#installing-prebuilt-dali-packages` instead of `#id1` I think.

Reply:
is it? Doesn't seem so in the docs we have published

Reply:
That is the parent section, I'll use that one